pubsub/google/cloud/pubsub_v1/publisher/exceptions.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+from google.api.core.exceptions import GoogleAPICallError
+from google.api.core.exceptions import GoogleAPIError
+
+
+class PublishError(GoogleAPICallError):
+    pass
+
+
+class TimeoutError(GoogleAPIError):
```

Comment:
`concurrent.futures.TimeoutError` exists on 2.x by virtue of the `futures` package. It's the top-level `TimeoutError` that (unfortunately) doesn't exist in 2.x.